<launch>
	<node name="ai_robot_watcher" pkg="ai_robot_status" type="ai_robot_status_node.py" output="screen" required="True"/>
</launch>